polkadot-protocol/protocol-components/accounts/account-data-structure.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Account Data Structure
+description: TODO
+---
+
+# Account Data Structure
+
+!!! warning "Update Notice"
```

Comment:
Not sure if we need to keep this in these docs?

Reply:
I think we should remove this